.github/workflows/demo.yaml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do
+# - a clean install of node dependencies
+# - run tests
+# - build and upload the storybook to a staging area
+#
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+name: Deploy
+jobs:
+    deploy:
+        runs-on: ubuntu-latest
+        if: ${{ github.repository_owner == 'wmde' }}
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Set up node
+              uses: actions/setup-node@v2
+              with:
+                  node-version: "16"
+            - name: Install dependencies
+              run: npm ci
+            - name: Build Project as app
+              run: npm run buildPreview
+            - name: Deploy to netlify
+              id: netlify_deploy
+              run: npx -p netlify-cli
```

Comment:
Do we need the `-p netlify-cli` when it should already be installed as part of the `npm ci` earlier?

Reply:
Good point. I wonder if could go in the opposite direction though? If we add `-p netlify-cli` also to the other command, maybe we can drop the dev dependency on Netlify and thus get rid of those vulnerability notices. That would seem to be a good trade-off given that most of use will likely not use the manual deploy much, since we don't even have the needed credentials.

Reply:
I’d be quite happy with that, it should also make `npm install` much faster. @guergana WDYT?

Reply:
If we dont need to install it locally then we shouldn't do it. go for it! :)